### ISSUE TITLE
Periodic republishing of key-value pairs causes TTL expiration deletion to fail?

### ISSUE THREAD

**Issue by kkisbug** [bmuller/kademlia#104]
I noticed that the logic to republish key-value pairs every hour seems to refresh the timestamp of the key-value pairs (via the _refresh_table method in network.py), thereby causing the TTL expiration deletion logic for the key-value pairs to become ineffective (if the expiration time is longer than the republishing interval). Is this intentional?

**Comment by bmuller** [comment]
That's correct - but only for the keys for which the current node is one of the closest.  If other nodes come online that are closer than those items in the current node's storage will eventually be forgotten.
